### PR TITLE
Mr add review modal

### PR DESCRIPTION
**Pull request by merosenlund**
- Created a ReviewForm component
- Imported Yosef's Modal CSS
- Made so ReviewForm renders when ADD A REVIEW + is clicked and disappears when the CLOSE button is clicked

[Trello Ticket](https://trello.com/c/aner7LRi)

Functionality Added

### CODE REVIEW

**Comment by merosenlund** [comment]
[Make modal toggle when Add Review is clicked or the close modal button is clicked](https://trello.com/c/aner7LRi/168-make-modal-toggle-when-add-review-is-clicked-or-the-close-modal-button-is-clicked)

File: src/components/reviews/ReviewsList.jsx [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
   const reviewsList = reviews.results;
 
+  const startReview = () => {
+    setModalState(true);
+  }
+
+  const cancelReview = () => {
+    setModalState(false);
+  }
+
```
Comment (DevonL1010) [inline src/components/reviews/ReviewsList.jsx:20]: to DRY the code, maybe you can combine the startReview() and cancelReview() together :

const handleModalActive = () => {
        setModalState(!modalIsActive)
}

File: src/components/reviews/ReviewsList.jsx [hunk 2]
```diff
@@ -24,8 +35,9 @@ const ReviewsList = () => {
       </div>
       <div>
         <button>MORE REVIEWS</button>
-        <button>ADD A REVIEW +</button>
+        <button onClick={startReview}>ADD A REVIEW +</button>
       </div>
+      {modalIsActive && <ReviewForm close={cancelReview}/>}
```
Comment (DevonL1010) [inline src/components/reviews/ReviewsList.jsx:40]: {modalIsActive? <ReviewForm handleModalActive={handleModalActive}/> : <></>}

Comment (DevonL1010) [reply src/components/reviews/ReviewsList.jsx:40]: It's a little suggestion , and overall looks good to me.

**Pull request merged**